lib/gui/app/modules/progress-status.js

```diff
@@ -61,11 +62,13 @@ exports.fromFlashState = (state) => {
   } else if (isFlashing) {
     // eslint-disable-next-line no-eq-null
     if (state.percentage != null) {
-      return `${state.percentage}% Flashing`
+      const txt = (plain) ? `${state.percentage}%` : `<strong>${state.percentage}%</strong>`
+      return `Flashing... ${txt}`
     }
     return `${units.bytesToClosestUnit(state.position)} flashed`
   } else if (isValidating) {
-    return `${state.percentage}% Validating`
+    const txt = (plain) ? `${state.percentage}%` : `<strong>${state.percentage}%</strong>`
```

Comment:
We should avoid html tags in strings.
Looks like we need an `FlashProgressLabel` component.